feat(Stormshield): create parser (#43)
SEKOIA-IO/intake-formats: Stormshield/README.md

# Stormshield Network Security

## Description

> The Stormshield Network Security firewall range provides the essential functionality you need to meet your business challenges. They are backed by trusted products that are certified to the highest European standards.
> 
> With this parser enabled, each event encountered by your Stormshield firewall will be available in SEKOIA.IO.

Reviewer: ```suggestion
Stormshield Network Security is a range of network security appliances.

With this parser enabled, each event encountered by your Stormshield firewall will be available in SEKOIA.IO.
```
